config/initializers/devise.rb

```diff
@@ -239,41 +238,78 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
-  omniauth_config = (File.exist? Rails.root.join('config', 'omniauth.yml')) && (Rails.application.config_for :omniauth)
```

Comment:
so config/omniauth.yml is not needed / can be removed then ?

Reply:
yes, you can remove them from the server.